apps/api/v1/pages/api/bookings/_post.ts

```diff
@@ -47,6 +47,10 @@ import { getAccessibleUsers } from "~/lib/utils/retrieveScopedAccessibleUsers";
  *                 type: string
  *                 format: date-time
  *                 description: 'End time of the Event'
+ *               rescheduleUid:
+ *                 type: string
+ *                 format: UID
+ *                 description: 'Uid of the booking to reschedule'
```

Comment:
rescheduling was already possible, just not specified in the doc

Reply:
@alishaz-polymath will this update the doc ?

Reply:
For V1, no because we do it in the [apiSpec](https://github.com/calcom/docs/blob/main/components/components/lib/apiSpec.tsx) in the docs repo.